### DISCUSSION TITLE
background-gradient & background-parallax

### DISCUSSION THREAD

**Discussion by vikas-growisto** [fastn-stack/fastn#586] [Q&A]
```
-- ftd.row:
width.fixed.px: 400
height.fixed.px: 200
gradient-direction: left to right
gradient-colors: red , blue

-- end: ftd.row
```

The property is not working can you provide an example that is working with the property.
need background-parallax example.

**Comment by Arpita-Jaiswal** [comment]
`background-gradient` & `background-parallax` are not yet supported. Please subscribe to https://github.com/ftd-lang/ftd/issues/591 to get updates and notifications.
